forms/reefWatch/Underwater weeding survey.json

```diff
@@ -0,0 +1,512 @@
+{
+  "id": "668c7b7dac0eee3abb3cdf9a",
+  "dateCreated": "2024-07-09T10:04:03Z",
+  "minOptionalSectionsCompleted": 1,
+  "supportsSites": false,
+  "tags": [],
+  "lastUpdated": "",
+  "createdUserId": "56575",
+  "external": false,
+  "activationDate": null,
+  "supportsPhotoPoints": false,
+  "publicationStatus": "published",
+  "externalIds": null,
+  "gmsId": null,
+  "name": "Underwater weeding",
+  "sections": [
+    {
+      "collapsedByDefault": false,
+      "template": {
+        "modelName": "Underwater weeding",
+        "dataModel": [
+          {
+            "dataType": "text",
+            "name": "supervisor",
+            "indexName": "supervisor",
+            "description": "",
+            "validate": "required"
+          },
+          {
+            "dataType": "date",
+            "name": "eventDate",
+            "indexName": "eventDate",
```

Comment:
Good to keep eventDate in indexName since it can help filter/facet if user needs.

Reply:
Thanks @temi I've removed a few indexes.
I thought the indexes are requirement when making forms in biocollect.